recipes/ldas-tools-diskcacheapi-swig/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
I think you need this to make sure it fails when there is an error
```suggestion
set -e

```